### DISCUSSION TITLE
how to access border_radius

### DISCUSSION THREAD

**Discussion by Razer-0** [beeware/toga#2524] [Q&A] [answered]
how to access css properties like border_radius
```
myImage = toga.Image('resources/callwave.ico')
		imagen = toga.ImageView(myImage)
		imagen.style.update(padding=20,  # Agrega relleno para mejorar la apariencia
							background_color='blue',  # Cambia el color de fondo de la imagen
							border_radius=20)
```

This marks an error in border_radius

**Comment by freakboy3742** [comment]
Toga's style language is very *similar* to CSS, but it is not CSS. 

The full specification for Pack, including all the style directives that are legal, can be [found here](https://toga.readthedocs.io/en/stable/reference/style/pack.html#pack-style-properties).

`border_radius` is not a supported property.

**Reply by Razer-0** [reply]
hello @freakboy3742 thanks for replying, is there any alternative way I can interact with the border of a `toga.Box()` or `toga.Button()`, I need to make my application not look so square so to speak some way.

**Reply by freakboy3742** [reply, marked as answer]
There is no way to add "rounded corners to buttons", or make similar style changes. This is by design, and is unlikely to change.

Toga is a platform-native toolkit. It uses system native buttons to make the buttons in your app look like your operating system's button style. If you don't like how your operating system renders buttons, either take that up with your operating system vendor, or use a different GUI toolkit that doesn't use native system widgets (e.g., Kivy).

**Reply by Razer-0** [reply]
I understand, thank you

**Reply by leonid1123** [reply]
is there any way to accsess border-color?

**Reply by freakboy3742** [reply]
There's no support for borders - so no, there's no support for border-color.

**Reply by mhsmith** [reply]
For discussion of borders, see #849.
